BiweeklyMeetings/2022-08-02.md

```diff
@@ -0,0 +1,323 @@
+
+**Sebastien** -: Yeah. So for this CIP, as you mentioned, is kind of a solid, better version of 25. The problem of 25, of courses is that I have to rely on this initial mint transaction and there is no guarantees that this is the actual mint. This one is slightly better, but still has the same issue, which is that now it's including stuff as part of the asset name. And now you have no way to know if the asset name is really by design or if this was a smart contract where users got to pick the asset name, this is a bit rarer of a situation. So it's slightly better. I still think it's slightly problematic. Personally, I prefer the option that [inaudible 00:44:11] maker came up with. So if there's another CIP that's the DID standard for Cardano and inside, they did standard. They did something very similar to this metadata standard, except they added another layer of indirection to avoid having to have the asset name stuffing, which I think is probably the better approach, but all things considered, I don't think there's anything wrong with CIPP the way it stands now. Obviously there's some issues with it as I mentioned, but if people use it, understanding of these issues, then I don't see anything wrong with it in particular.
+
+**Matthias** -: Okay. We haven't really looked at CIP 66. So this did stand up proposed by the team, which might be good then to bring on the table, at least in the next meeting or next to the new metadata discussion, especially if they have some conflicting opinions on these.
+
+**Sebastien** -: Yeah, I think there were three PRs with conflicting implementations of this many data standard. So there's the metadata standard for us as the end maker one. Then there's a third one that also had something similar. I don't remember.
+
+**Matthias** -: Well, there was on chain metadata proposed by this web team a while back, but I think this one got...
+
+**Sebastien** -: Cardano Smart NFTs.
+
+**Matthias** -: Oh no, that was yet another one, the smart NFTs. The smart NFT were about embedding actual codes on the blockchains that makes those NFT... I mean, that's an API for those NFTs to be embedded.
+Speaker 4:
+Right. And they want the way to upgrade these...
+
+**Matthias** -: Yeah. To make them [inaudible 00:46:06] and interactive in a way. But there is some overlap maybe less than with CIP six. Okay. So that would be important to mention and to see if we can get maybe both offers to discuss where there is no problem in having two different proposals. That's always a pain down the line. So platform that need to implement both of them as that both gets really adopted. So we might as well want to have only one. So I'll mention that on the request.
```

Comment:
```suggestion
**Matthias** -: Yeah. To make them programmable and interactive in a way. But there is some overlap maybe less than with CIP 66. Okay. So that would be important to mention and to see if we can get maybe both offers to discuss where there is no problem in having two different proposals. That's always a pain down the line. So platform that need to implement both of them as that both gets really adopted. So we might as well want to have only one. So I'll mention that on the request.
```